Sources/Kitura/RouterRequest.swift

```diff
@@ -234,42 +225,41 @@ public class RouterRequest {
 }
 
 private class Cookies {
-
-    /// Storage of parsed Cookie headers
-    fileprivate var cookies = [String: HTTPCookie]()
-
-    /// Static for Cookie header key value
-    private let cookieHeader = "cookie"
-
-    fileprivate init(headers: HeadersContainer) {
-        guard let rawCookies = headers[cookieHeader] else {
-            return
+    fileprivate static func parse(headers: HeadersContainer) -> [String: HTTPCookie] {
+        var cookies = [String: HTTPCookie]()
+        guard let cookieHeaders = headers["cookie"] else {
+            return cookies
         }
-        for cookie in rawCookies {
-            initCookie(cookie, cookies: &cookies)
+
+        for cookieHeader in cookieHeaders {
+            for cookie in cookieHeader.components(separatedBy: ";") {
+                let trimmedCookie = cookie.trimmingCharacters(in: .whitespaces)
```

Comment:
The standard for the separator is semicolons with zero or more whitespace. Shmuel suggested removing the regex, splitting by semicolons and stripping out leading/trailing whitespace, which achieves the goal without using regular expressions.